docs/index.md

```diff
@@ -147,3 +147,4 @@ There are a few things that can be checked if you review a pull request against
 * Are there new params with datatype Hash or Array? If possible, they should default to empty Hash/Array instead of undef. You can also enforce the datastructure like Array[String]
 * Are there new params with datatype Boolean? The default value is a tricky decision which needs careful reviewing. Sometimes a True/False is the better approach, sometimes undef
 * Is this a bugfix? Write the Pull Request Title in a way that users can easily identify if they are impacted or not
+* Is a new template added? The preferred language is [epp](https://puppet.com/docs/puppet/5.5/lang_template_epp.html), not [erb](https://puppet.com/docs/puppet/5.5/lang_template_erb.html)
```

Comment:
I would use https://puppet.com/docs/puppet/latest/lang_template_epp.html instead of https://puppet.com/docs/puppet/5.5/lang_template_epp.html. Same is for the erb link.